src/core/os/amdgpu/amdgpuDevice.cpp

```diff
@@ -2119,6 +2123,23 @@ bool Device::HasFp16DisplaySupport()
     return supported;
 }
 
+// Returns true if gpu + amdgpu kms driver do support 16 bit unorm fixed point display.
```

Comment:
Oh, you also need a comment right above this one that's just this line, as a header:
// =====================================================================================================================

That comment is also missing from above HasFp16DisplaySupport().

Reply:
Ok. Thanks for the feedback.